### PR TITLE
execute docker commands inside container

### PR DESCRIPTION
**Pull request by ilya-varchenya**

### CODE REVIEW

**Comment by ilya-varchenya** [comment]
*make available docker inside jenkins-slave container

File: docker-compose.yml [hunk 1]
```diff
@@ -70,6 +71,7 @@ services:
      - JENKINS_SLAVE_LABELS=qps-slave api qa
```
Comment (vdelendik) [inline docker-compose.yml]: add "docker" as a label to jenkins-slave-api

File: docker-compose.yml [hunk 2]
```diff
@@ -57,6 +57,7 @@ services:
      # to add extra plugins follow this steps: https://github.com/qaprosoft/jenkins-master/releases/tag/4.6
      - ./jenkins:/var/jenkins_home
      - $HOME/.m2:/var/jenkins_home/.m2
+     - /var/run/docker.sock:/var/run/docker.sock
```
Comment (vdelendik) [inline docker-compose.yml]: remove docker.sock sharing for web slave. keep it only for api where we add explicit docker label

File: docker-compose.yml [hunk 3]
```diff
@@ -85,6 +87,7 @@ services:
      - JENKINS_SLAVE_LABELS=qps-slave web qa
     volumes:
      - $HOME/.m2:/root/.m2
+     - /var/run/docker.sock:/var/run/docker.sock
```
Comment (vdelendik) [inline docker-compose.yml]: remove share

**Pull request merged**